lib/Dialect/FIRRTL/Transforms/EmitOMIR.cpp

```diff
@@ -253,6 +253,16 @@ static std::optional<Attribute> scatterOMIR(Attribute original,
           tracker.append("id", idAttr);
           tracker.append("target", StringAttr::get(ctx, value));
 
+          // Add the dontTouch annotation to the target for
+          // OMDontTouchedReferenceTarget.
+          if (tpe == "OMDontTouchedReferenceTarget") {
```

Comment:
Is this too late?  Shouldn't this be added in LowerAnnotations?
Even better, shouldn't the scala be changed so that this doesn't exist and a don't touch is used in the first place?

Reply:
This function is used in LowerAnnotations so I think this is appropriate place. For Chisel change, yes that looks better and I'll take a look.